torch/_dynamo/variables/torch_function.py

```diff
@@ -45,6 +46,24 @@ def is_torch_function_user_object(obj):
     return hasattr(obj, "__torch_function__")
 
 
+def _is_attr_overidden(tx, var, name):
+    import torch
+
+    try:
+        attr_val = inspect.getattr_static(var.python_type(), name)
+        return attr_val != getattr(torch.Tensor, name)
+    except AttributeError:
+        pass
+
+    try:
+        var.dynamic_getattr(tx, name)
```

Comment:
Do we need to guard on this?

Reply:
I removed the dynamic_getattr piece because it was giving some false positives. I added guards to ensure the an attribute/method/property is not overridden though if we make it past this check.